calculatorApi1.js

```diff
@@ -0,0 +1,51 @@
+const express = require("express");
+const app = express()
+const cors = require("cors")
+app.use(cors())
+const port = 3000
+
+const add = (num1, num2) =>  num1 + num2; 
+
+const sub = (num1, num2) => num1 - num2 
+
+const mul = (num1, num2) => num1 * num2
+
+const div = (num1, num2) => num1 / num2
+
+
+app.get("/", (req, res) => {
+    res.send("You made a get request at / url")
+})
+
+
+app.get("/add", (req, res) => {
+    res.json(add)
+})
+
```

Comment:
Right now you're sending up a uninvoked function. You'll need to invoke it and pass in the arguments.